src/_atomic_data.cpp

```diff
@@ -18,25 +17,6 @@ void pyne::_load_atomic_mass_map_memory() {
   } else { 
     _insert_abund_map();
   }
-
-  // calculate the atomic_masses of the elements
-  std::map<int,double> :: iterator it;
-
```

Comment:
I think because it was replaced with the `pyne::_insert_atomic_mass_map()` function below, which gets called above in this function.  There is no longer any reason to read in this data from disk.